packages/base-types/src/models/shared.ts

```diff
@@ -54,7 +54,7 @@ export type Intent = {
   name: string;
   slots?: IntentSlot[];
   inputs: IntentInput[];
-  builtIn: boolean;
+  builtIn?: boolean;
```

Comment:
reasoning for this?

Reply:
Missed when converting from superstruct objects to types